syscalls_bsd.go

```diff
@@ -0,0 +1,45 @@
+// +build openbsd freebsd
+
+package water
+
+import (
+	"os"
+)
+
+const bsdTUNSIFINFO = (0x80000000) | ((4 & 0x1fff) << 16) | uint32(byte('t'))<<8 | 91
```

Comment:
If this is from a BSD source, please kindly mention the original header name and quote some important definitions, so that it's easier to understand if somebody else needs to change this.

Reply:
I will look to add appropriate comments for it.